### DISCUSSION TITLE
The result of compression is same with or not using Keka apps

### DISCUSSION THREAD

**Discussion by Hex0-sys** [aonez/Keka#1144] [Q&A] [answered]
Hello. recently i'm purchased your apps and i'm wondering why the result of size is same when i am using or not using Keka apps? 
I think when i'm using your apps, when i want to compress something it will be a smaller size than a "default compress" from Mac. Can you solved this issue? or this is normal? 🥲😔
Here is a proof:
![Screen Recording 2022-07-16 at 22 28 29](https://user-images.githubusercontent.com/100767432/179361675-2fb9872f-6349-4de3-bc2d-95de7836c49a.gif)

Thank you :)

**Comment by aonez** [comment, marked as answer]
The resulting size depends on the compression that can be achieved that depends on the source files. Try for example compressing the "Safari.app" in the Applications folder, you'll see differences there.

Also check the size with the "Get Info" panel that displays the size in bytes.

**Reply by Hex0-sys** [reply]
Ok i see, thank you.

**Comment by gingerbeardman** [comment]
Also, using 7z rather than zip will yield smaller results, or by changing the settings for each of those types to "smaller".

**Reply by Hex0-sys** [reply]
Thank's for the advice
